# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/gnome2/memprof/cooked-size-newbinutils.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./bfdutils.c.orig	2004-09-07 18:10:27.670831536 +0200
+++ ./bfdutils.c	2004-09-07 18:10:39.396049032 +0200
@@ -164,7 +164,7 @@
 				 * into the internals instead. :-(
 				 */
 				/* (*func) (addr, bfd_section_size (map->abfd, section), user_data); */
-				(*func) (addr, section->_cooked_size, user_data);
+				(*func) (addr, section->size, user_data);
 			}
 		}
 }
